test/e2e/helpers_test.go

```diff
@@ -291,3 +292,45 @@ func killKong(ctx context.Context, t *testing.T, env environments.Environment, p
 	}, kongComponentWait, time.Second)
 	t.Logf("kong container has %v restart after kill", after)
 }
+
+// buildImageLoadAddons creates addons to load KIC and kong images.
+func buildImageLoadAddonss(t *testing.T, images []string) []clusters.Addon {
+	addons := []clusters.Addon{}
+	for _, image := range images {
+		if image != "" {
+			t.Logf("load image %s", image)
+			b, err := loadimage.NewBuilder().WithImage(image)
+			require.NoError(t, err)
+			addons = append(addons, b.Build())
+		}
+	}
+	return addons
+}
+
+func createKongImagePullSecret(ctx context.Context, t *testing.T, env environments.Environment) {
```

Comment:
This would use a comment. Since it has no return values it would be ideal to document the side effects.

Reply:
added comments.